x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/ssl/SSLReloadDuringStartupIntegTests.java

```diff
@@ -167,8 +165,8 @@ private void waitUntilFixKeystoreIsReadyToBegin(final CountDownLatch beforeKeyst
         });
     }
 
-    static void timed(final Logger LOGGER, final Level level, final String message, final Runnable runnable) {
-        assert LOGGER != null;
+    static void timed(final Logger logger, final Level level, final String message, final Runnable runnable) {
```

Comment:
What's going on with this variable? Still needed?

Reply:
Hmm guess it never was needed actually, just made this a private non-static instance method now so we don't need that param :)